core/src/test/scala/unit/kafka/log/remote/RemoteLogManagerTest.scala

```diff
@@ -413,14 +413,20 @@ class RemoteLogManagerTest {
     epochCheckpoints.foreach { case (epoch, startOffset) => cache.assign(epoch, startOffset) }
     val currentLeaderEpoch = epochCheckpoints.last._1
 
+    val localLogSegmentsSize = 500L
```

Comment:
Can you help to improve the test readability? We have added a random `localLogSegmentsSize` value as 500 to trigger the remote log segment deletions. 

Can we have a new variable for segment size?
```
val segmentSize = 1024
val overlappingSegments = 3
val overlappingLogSegmentsSize = overlappingSegments * segmentSize
val localLogSegmentsSize = 512 + overlappingLogSegmentsSize
val retentionSize = ((segmentCount - deletableSegmentCount) * segmentSize) + (localLogSegmentsSize - overlappingLogSegmentsSize)
...
val localLogStartOffset = recordsPerSegment * segmentCount - (overlappingSegments * segmentCount)

With overlapping segments, we also have to adjust the input test case.